Localization/en-US.hjson

```diff
@@ -22,56 +16,86 @@ Mods: {
 		SwitchToNormal: Switch to Normal Mode
 		SwitchToExpert: Switch to Expert Mode
 		SwitchToMaster: Switch to Master Mode
-		DifficultyCommandUsage: Usage: /Difficulty <0(Normal)|1(Expert)|2(Master)>
+		DifficultyCommandUsage: Usage: /difficulty <0(Normal)|1(Expert)|2(Master)>
 
+		Items: {
+			FakeDemonHeart: {
+				DisplayName: Fake Demon Heart
+				Tooltip: ""
+			}
+		}
 
-		Config: {
-			$parentVal: Settings
-			ServerBlocked: Only the server owner can change the settings.
+		Configs: {
+			Config: {
+				DisplayName: Settings
+				ServerBlocked: Only the server owner can change the settings.
 
-			AddCheatSheetButton: {
-				$parentVal: Add Cheat Sheet/HERO's Mod button
-				Desc: Add a button that toggles difficulty mode to the Cheat Sheet mod and HERO's Mod.
-			}
+				CurrentFakedDifficulty: {
+					Label: Current Faked Difficulty
+					Tooltip: Makes the enabled settings think it is set to this difficulty.
+				}
 
-			ChangeAI: {
-				$parentVal: Change Boss AI
-				Desc:
-					'''
-					Bring Expert Mode boss AI into Normal Mode.
-					(Warning: may break some modded bosses)
-					'''
-			}
+				DropTreasureBagsInNormal: {
+					Label: Modify loot
+					Tooltip: Changes loot in accordence with your selected Beneficial Difficulty.
+				}
 
-			DropBags: Treasure Bags in Normal Mode
+				AddCheatSheetButton: {
+					Label: Add Cheat Sheet/HERO's Mod button
+					Tooltip: Add a button that toggles difficulty mode to the Cheat Sheet mod and HERO's Mod.
+				}
 
-			ExpertCommand: {
-				$parentVal: Add /difficulty command
-				Desc: Add the /difficulty chat command that toggles Expert Mode.
-			}
+				ChangeBossAI: {
+					Label: Change Boss AI
+					Tooltip:
+						'''
+						Bring Expert Mode boss AI into Normal Mode.
+						(Warning: may break some modded bosses)
+						'''
+				}
 
-			CurrentFakedDifficulty: {
-				$parentVal: Current Faked Difficulty
-				Desc: Makes the world think it is set to this difficulty.
-			}
+				SlotsWorksInNormal: {
+					Label: Accessory Slots
+					Tooltip: The extra accessory slot(s) work based on the current beneficial difficulty.
+				}
 
-			SlotsHack: {
-				$parentVal: Accessory Slots
-				Desc: The extra accessory slot(s) work in Normal Mode.
-			}
+				BossBlacklist: {
+					Label: Boss Blacklist
+					Tooltip: These NPCs won't be affected by Boss Expertise.
+				}
 
-			BossBlacklist: {
-				$parentVal: Boss Blacklist
-				Desc: These bosses won't be affected by Boss Expertise.
-			}
+				BossWhitelist: {
+					Label: Boss Whitelist
+					Tooltip:
+						'''
+						These NPCs will be affected by Boss Expertise
+						even if they're technically not bosses.
+						'''
+				}
+
+				FakedBeneficialDifficulty: {
+					Label: Faked Beneficial Difficulty
+					Tooltip: ""
+				}
+
+				ForTheWorthy: {
+					Label: For The Worthy
+					Tooltip: ""
+				}
```

Comment:
Do we need to comment this out it enlarges the file by X bytes but it doesn't hurt since it is unused since I commented out the code. I'll will let you decide it isn't really important.

Reply:
Well I saw the localization string and then went searching for where it's used. Took me a while to realize it's unused. So I decided to make this more explicit.